app/scripts/modules/core/src/domain/IArtifact.ts

```diff
@@ -1,13 +1,14 @@
 export interface IArtifact {
-  kind?: string; // json model only
   id: string;
   type?: string;
-  customKind?: boolean;
   name?: string;
   version?: string;
   location?: string;
   reference?: string;
   metadata?: any;
   artifactAccount?: string;
   provenance?: string;
+  // Legacy artifacts properties
+  kind?: string; // TODO delete
+  customKind?: boolean; // TODO delete
```

Comment:
Is the intention in this PR to remove custom kinds or not? There are lots of places where it looks like the functionality has been pulled out but others where it looks like we're trying to maintain it. I would like to see them remain until 1.13's release so that there is ample time to give clear notice to anyone who is currently using them.

Reply:
just indicating that customKind will be removed with 1.14. not aware of anywhere that previously referenced customKind that doesn't now...